package.json

```diff
@@ -91,7 +99,7 @@
     "audit-client": "npm-run-all -p -l audit-client-*",
     "lint-client-template": "cd sdk/template/template && npm run eslint",
     "lint-client": "npm-run-all -p -l lint-client-*",
-    "analyze-client": "npm run audit-client && npm run lint-client"
+    "analyze-client": "npm run audit-client && npm run lint-client && npm run check-format-client"
```

Comment:
One concern of making it part of the build is that people might forget to format and got failures in build pipeline. We could add it as to the npm build script of the package but @mikeharder prefer separating analyzing from building. We could do fancier things like formatting in precommit.

Reply:
The engineering system team (@Azure/azure-sdk-eng) has decided, at least for now, that incorrect style should **not** cause build failures (even in the `analyze` step).  We feel the cost of extra build failures outweighs the benefit of 100% consistent style.  It's sufficient for the repo to be configured so most editors will use the correct style by default.

It's fine to keep the `format-*` and `check-format-*` scripts if you want them to be available for developer inner loop, but `analyze-client` should not run `check-format-client`.

Reply:
Makes sense. Reverted this.